### PR TITLE
Issue #2624874 by Primsi: Provide a render element

### PR DESCRIPTION
**Pull request by primsi**

### CODE REVIEW

**Comment by primsi** [comment]
Work in progress atm.

File: src/Element/EntityBrowserElement.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+   */
+  public static function valueCallback(&$element, $input, FormStateInterface $form_state) {
+    $return = [];
+    return $return;
+  }
+
+  /**
+   * AJAX form callback.
+   */
+  public static function updateWidgetCallback(array &$form, FormStateInterface $form_state) {
```
Comment (slashrsm) [inline src/Element/EntityBrowserElement.php:117]: This is field widget specific function. Should not be here.

File: src/Element/EntityBrowserElement.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    $element['#variables']['entity_browser'] = $element['entity_browser'];
+
+    return $element;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function valueCallback(&$element, $input, FormStateInterface $form_state) {
+    $return = [];
```
Comment (slashrsm) [inline src/Element/EntityBrowserElement.php:110]: return [];

File: src/Element/EntityBrowserElement.php [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+   */
+  public static function preRenderEntityBrowser($element) {
+    $hidden_id = Html::getUniqueId('edit-' . $element['#identifier'] . '-target-id');
+    $details_id = Html::getUniqueId('edit-' . $element['#identifier']);
+
+    $element['#theme_wrappers'][] = [
+      '#id' => $details_id,
+      '#type' => 'details',
+      '#open' => !empty($ids),
+      'target_id' => [
```
Comment (slashrsm) [inline src/Element/EntityBrowserElement.php:85]: target_id is field widget specific thing.

Comment (slashrsm) [inline src/Element/EntityBrowserElement.php:85]: And same with wrapper I guess.

**Comment by dawehner** [comment]
We have this render element already, right? I think this PR is no longer needed.

**Comment by slashrsm** [comment]
True.

**Pull request closed without merging**